Packs/AzureKeyVault/Integrations/AzureKeyVault/AzureKeyVault_description.md

```diff
@@ -0,0 +1,36 @@
+ ### Authorize Cortex XSOAR for Azure Key Vault (self-deployed configuration)
+Follow these steps for a self-deployed configuration.
+
+
+1. To use a self-configured Azure application, you need to add a new Azure App Registration in the Azure Portal. To add the registration, refer to the following [Azure app registration article](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app)
+
+
+2. You must grant to YOUR-REGISTERED-APP permissions, for your Azure Key vault. 
```

Comment:
Same here - let's review the grammar and phrasing of this whole description. It can be improved.

Reply:
Done.